src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -432,7 +432,9 @@ private EditorFormFieldSet generateEditorFormFieldSet(Set<String> processedPrope
             EditorFormField editorFormField = generateEditorFormField(itemDefinition, existingNode, parentNode, uiLocale, locale, rank);
 
             editorFormFields.add(editorFormField);
-            processedProperties.add(itemDefinition.getName());
+            if (!dynamic) {
```

Comment:
so you will only add not dynamic fieldset (mixin), right?

Reply:
yes, that's the point to allow several mixins to add the same property.